<launch>
    	<node name="guidanceFrontCameras" 	type="guidanceFrontCameras" pkg="guidance" output="screen" />
    	
	<param name="front_cam_left_file" 	value="$(find guidance)/calibration_files/1/left.yaml" />
	<param name="front_cam_right_file" 	value="$(find guidance)/calibration_files/1/right.yaml" />
	
	<arg name="pi/2" value="1.5707963267948966" />
	<arg name="pi" value="3.141592654" />
	
	<node pkg="tf2_ros" type="static_transform_publisher" name="robot_guidanceFront_link" 	args="0.10 0 0.075 	-$(arg pi/2) 0 -$(arg pi/2) 	base_link guidanceFront" />
		
	<node name="guidanceFront" 	ns="/guidance/1" pkg="stereo_image_proc" type="stereo_image_proc" output="screen" />
</launch>
<!-- TODO nodes, if launched, don't run somehow.  -->
